src/patternfly/components/Table/table-td.hbs

```diff
@@ -1,13 +1,15 @@
 <{{#if table-td--type}}{{table-td--type}}{{else}}td{{/if}} class="{{pfv}}table__td
-  {{~#if table-td--toggle}} {{pfv}}table__toggle{{/if}}
+  {{~#if table-td--IsToggle}} {{pfv}}table__toggle{{/if}}
   {{~#if table-td--check}} {{pfv}}table__check{{/if}}
-  {{~#if table-td--action}} {{pfv}}table__action{{/if}}
+  {{~#if table-td--IsAction}} {{pfv}}table__action{{/if}}
+  {{~#if table-td--HasOverflowMenu}} {{pfv}}table__action{{/if}}
```

Comment:
This probably needs to be updated to `table-td--UsesOverflowMenu` as that is being used elsewhere. Currently nothing is being rendered in the last column of the overflow table examples.